educator/src/Dscp/Educator/Web/Student/Auth.hs

```diff
@@ -44,19 +49,28 @@ newtype GetStudentsAction = GetStudentsAction (IO [PublicKey])
 -- Instances
 ---------------------------------------------------------------------------
 
-instance ToJWT (WithStudent a) where
-    -- `servant-auth` insists on putting a JWT of this type into cookie headers
-    -- and I couldn't find a way to disable that.
-    -- Since making a ToJWT instance for this type doesn't make sense, but is
-    -- required by `servant-auth` we just placeholder it with an empty claims.
-    -- Note: this instance should *never* be used outside of what servant-auth
-    -- is using it for: setting the cookie headers.
-    encodeJWT = const emptyClaimsSet
+instance ( HasServer api ctxs
+         , HasContextEntry ctxs GetStudentsAction
+         ) => HasServer (Auth' :> api) ctxs where
+    type ServerT (Auth' :> api) m = Student -> ServerT api m
+
+    hoistServerWithContext _ pc nt s = hoistServerWithContext (Proxy :: Proxy api) pc nt . s
+
+    route _ context subserver =
+      route (Proxy :: Proxy api)
```

Comment:
Can you please align these three lines so they look like `route` takes two arguments and not like you just forgot to add `do`.